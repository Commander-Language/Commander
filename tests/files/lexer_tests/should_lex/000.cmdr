// Test that a simple command lexes properly
echo "Hello World!";