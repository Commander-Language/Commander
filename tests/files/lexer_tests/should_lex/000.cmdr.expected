CMDSTRINGVAL 'echo' 2:1
STRINGVAL '"' 2:6
STRINGLITERAL 'Hello World!' 2:7
ENDSTRINGVAL '"' 2:19
SEMICOLON ';' 2:20
END_OF_FILE 2:21
